.azure-pipelines/azure-pipelines-osx.yml

```diff
@@ -0,0 +1,92 @@
+# This file was generated automatically from conda-smithy. To update this configuration,
+# update the conda-forge.yml and/or the recipe/meta.yaml.
+# -*- mode: yaml -*-
+
+jobs:
+- job: osx
+  pool:
+    vmImage: macOS-10.13
+  timeoutInMinutes: 240
+  strategy:
+    maxParallel: 8
+    matrix:
+      osx_python2.7:
+        CONFIG: osx_python2.7
```

Comment:
Why are we building for 2.7?

Reply:
The azure builds are not running at all which I don't understand why are they defined at all.

It might be because We're not using selectors, just constraints for `py>3`. I can't find the conda issue, but the noarch py3k packages are supposed to be defined like @xmnlab did.